pdns/recursordist/docs/upgrade.rst

```diff
@@ -7,8 +7,42 @@ When upgrading several versions, please read **all** notes applying to the upgra
 4.2.x to 4.3.0 or master
 ------------------------
 
+Lua Netmask class methods changed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - Netmask class methods ``isIpv4`` and ``isIpv6`` have been deprecated in Lua, use :func:`Netmask.isIPv4` and :func:`Netmask.isIPv6` instead. In C++ API these methods have been removed.
 
+``socket-dir`` changed
+^^^^^^^^^^^^^^^^^^^^^^
+The default :ref:`setting-socket-dir` has changed to include ``pdns-recursor`` in the path.
+For non-chrooted setups, it is now whatever is passed to ``--with-socketdir`` during configure (``/var/run`` by default) plus ``pdns_recursor``.
+The systemd unit-file is updated to reflect this change and systemd will automatically create the directory with the proper permissions.
+The packaged sysV init-script also creates this directory.
+For other operating systems, update your init-scripts accordingly.
+
+Systemd service and permissions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The systemd service-file that is installed no longer uses the ``root`` user to start.
+It uses the user and group set with the ``--with-service-user`` and ``--with-service-group`` switches during
+configuration, "pdns" by default.
+This could mean that PowerDNS Recursor cannot read its configuration or zone-file data.
```

Comment:
Should "or zone-file data" be changed or removed? Recursor does support RPZ, but still.

Reply:
I'll update to mention recursor relevant files.